Utils/extensionutils.py

```diff
@@ -200,7 +200,7 @@ def run_command_get_output(cmd, chk_err=True, log_cmd=True):
                 'CalledProcessError.  Error message is ' + str(e))
             return e.errno, str(e)
     # noinspection PyUnboundLocalVariable
-    return 0, output.decode('latin-1')
+    return 0, output.decode('utf-8')
```

Comment:
looks like other method(s) has this usage as well - run_send_stdin. I remember we were hitting that method for vmaccess. Also, in the current method too, it is still using latin-1? Shouldn't we handle these as well?

Reply:
updated all output.decode to utf-8